pack/README.md

```diff
@@ -0,0 +1,40 @@
+# Use Pack to build container images
+
+Build container images using [pack](https://buildpacks.io/docs/install-pack/) and [buildpacks](https://buildpacks.io/).
+
+## Usage
+
+This extension adds the `pack` function, which is used the same way as `docker_build` to allow Tilt
+to automatically build a container image with a known name. For instance the following example shows
+building an image using the default builder and having that image automatically deployed to Kubernetes.
+
+```python
+load('ext://pack', 'pack')
+
+pack('example-image')
+k8s_yaml('kubernetes.yaml')
+k8s_resource('example-deployment', port_forwards=8000)
+```
+
+The `pack` function takes two new arguments:
+
+* `path`: path to application directory, defaults to current working directory
+* `builder`: builder image, defaults to gcr.io/paketo-buildpacks/builder:bas
+
+For instance, if you want to use the Cloud Foundry builder instead of the default [paketo](https://paketo.io/) one,
+you would do the following:
+
+```python
+pack(
+  'example-image',
+  builder='cloudfoundry/cnb:tiny'
```

Comment:
Hi! I'm a contributor to the Paketo Buildpacks project. Excited to see this extension 😄. 

Tiny nitpick - it might make sense to remove references of the "Cloud Foundry" builders here, as the Paketo Project will be deprecating them in favor of the Paketo Builder images in some time (likely when the next pack release comes out). Perhaps this could change to something like `If you want to use a different Paketo builder` and reference `gcr.io/paketo-buildpacks/builder:tiny`.

By the way, our full builder list is available here - https://github.com/paketo-buildpacks/builder. 

>